Integrations/VulnDB/VulnDB.yml

```
@@ -465,14 +471,15 @@ script:
       secret: false
     - default: false
       defaultValue: '20'
-      description: Maximum number of entries to return. A high number of entries might effect performance.
+      description: Maximum number of entries to return. A high number of entries might
+        effect performance.
```

```suggestion
        affect performance.
```
Make this change wherever it's relevant.